remo/domain/dataset.py

```diff
@@ -386,19 +386,19 @@ def images(self, limit: int = None, offset: int = None) -> List[Image]:
         """
         return self.sdk.list_dataset_images(self.id, limit=limit, offset=offset)
 
-    def search(self, class_list: List[str], task: str):
+    def search(self, classes: Union[str, List[str]] = None, task: str = None):
```

Comment:
@vmanilo 

What is 
`classes: Union[str, List[str]] = None` ?

I have to spend few minutes just to read it out...